pkgs/forecast/package.nix

```diff
@@ -11,7 +11,7 @@
 }:
 
 rustPlatform.buildRustPackage rec {
-  pname = "cosmic-ext-forecast";
+  pname = "forecast";
```

Comment:
This does not match upstream crate name:

<https://github.com/cosmic-utils/forecast/blob/d3a0215e5493b2ac6fdeb8c752dfa8eac13c5f21/Cargo.toml#L2>

Reply:
It is how it is called in nixpkgs as of now, and I also think it is a unique name among the package repos

Reply:
Hmm, well I don't feel strongly enough to argue about consistency/upstream's intent. So I suppose this is fine